appveyor.yml

```diff
@@ -1,12 +1,21 @@
+---
+
 image: 
-  - Visual Studio 2019
-  - Ubuntu
   - macos
+  - Ubuntu2004
+  - Visual Studio 2019
 
 configuration: Release
 
 environment:
   APPVEYOR_YML_DISABLE_PS_LINUX: true
+
+install:
+- sh: |
```

Comment:
Do the `sh` commands only run on Linux and Macos?

Reply:
Yes (at least by default anyway). There may be a similar variable to `APPVEYOR_YML_DISABLE_PS_LINUX` to enable bash on windows. But yeah, this doesn't run on windows.